package.json

```diff
@@ -118,7 +120,7 @@
       }
     },
     "test": {
-      "command": "npm run version-check && npm run unittest && npm run servertest && npm run eslint && npm run stylelint",
+      "command": "npm run version-check && npm run unittest && npm run servertest && npm run flow:check && npm run eslint && npm run stylelint",
```

Comment:
Yeah, I like that. I think solving Flow errors first is better because sometimes test failures can be hard to decipher when you've made a programming mistake.